src/azure-cli/azure/cli/command_modules/backup/custom_wl.py

```diff
@@ -632,3 +632,10 @@ def _get_protected_item_instance(item_type):
     if item_type.lower() == "saphanadatabase":
         return AzureVmWorkloadSAPHanaDatabaseProtectedItem()
     return AzureVmWorkloadSQLDatabaseProtectedItem()
+
+
+def _check_map(item_type, item_type_map):
+    if item_type_map.get(item_type) is not None:
+        return item_type_map[item_type]
+    error_text = item_type + " is an invalid argument. " + str(list(item_type_map.keys())) + " are the allowed values."
+    raise CLIError(error_text)
```

Comment:
Please use [InvalidArgumentValueError](https://github.com/Azure/azure-cli/blob/dev/src/azure-cli-core/azure/cli/core/azclierror.py#L122) for better error classification. @houk-ms for awareness.